### ISSUE TITLE
UI Changes to Component Pages

### ISSUE THREAD

**Issue by emsaiz** [OER-WEB-TEAM/drupal-subtheme--nihod#90]
_**LOW PRIORITY**_

### Overview 
1. Not in love with the pill subheading design for components: "do use for," "don't use for," "usability guidance," and "accessibility."   
2. It makes sense for “Do” and “Don’t” to be presented side by side in columns or a table for easy comparison. However, I don’t think “Usability guidance” and “Accessibility” should be formatted as opposing columns, since they’re not contrasting concepts. 

[Accordion Component Page Mockup (Figma)
](https://www.figma.com/proto/4eIHgCmkB0JL2OnaBKaQIS/Design-System-Mockups?node-id=27-5081&viewport=732%2C-2150%2C0.28&t=CDmbddwDk9kPBoCX-0&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=27%3A5081&show-proto-sidebar=1)

### Proposed UI Iterations to Component Pages 
- [ ] Remove all pills 
- [ ] Add background color and top border to dos and donts 
- [ ] Stack usability guidance and accessibility sections
